www-cronrobot/app/views/home/index.html.erb

```diff
@@ -1 +1,176 @@
-welcome to CronRobot
\ No newline at end of file
+<section id="home">
+
+    <!-- The Navbar of the website, will contain the logo, and the links -->
+    <nav>
+      <a href="#home">
+        <img id="logo" alt="Logo" src="images/Big Logo.svg">
+      </a>
+      <div id="center-nav">
+        <a href="#features">Features</a>
+        <a href="#pricing">Pricing</a>
+        <a href="#faq">FAQ</a>
+        <a href="https://docs.cronrobot.io">Docs</a>
+      </div>
+      <a onclick="handleLogin()" class="center" id="dashboard">
+        <img src="images/icons/dashboard.svg">
+        <p class="text-medium">Dashboard</p>
+      </a>
+      <button class="openbtn" onclick="openNav()">☰</button>
+    </nav>
+
+    <h1 id="heading">
+      The world's leading uptime monitoring service<blue>.</blue>
+    </h1>
+
+    <p id="tagline">
+      Downtime happens even to the best of us. But it’s important to know it before customers are affected!
+    </p>
+
+    <a onclick="handleLogin()">
+      <button class="btn-start">
+        <p>GET STARTED NOW</p>
+        <img src="images/icons/arrow-right.svg">
+      </button>
+    </a>
+
+</section>
+
+<section id="features">
```

Comment:
if we could have one image of notification channels

![image](https://user-images.githubusercontent.com/8007722/130611020-c68f4ded-ca97-4d15-9c55-910e47b555b0.png)

Reply:
Sure, would do

Reply:
Done 👍